automation/requirements.yml

```diff
@@ -10,3 +10,7 @@ collections:
     version: "==1.27.0"
   - name: hetzner.hcloud
     version: "==4.2.2"
+  - name: ansible.windows
+    version: "==2.7.0"
+  - name: community.postgresql
+    version: "==3.10.2"
```

Comment:
```suggestion
```
This is not a requirement for this project.

Reply:
Is is in `patroni/tasks` and `consul/handlers`? Besides, I am getting errors for `ansible-lint` if these are not installed.

Reply:
`community.postgresql` is possible, but windows is unlikely to be supported)

Reply:
>  am getting errors for ansible-lint if these are not installed.

ok

Reply:
> am getting errors for ansible-lint if these are not installed.
we could add an exception to the linter instead of installing a windows component that we don't actually need.

Reply:
It's in the module, so 🤷️ Don't wanna discuss about it, this is your expertise area. Just stating that these calls exists, which is why I've added them.